types/lodash/lodash-tests.ts

```diff
@@ -7076,9 +6809,9 @@ fp.now(); // $ExpectType number
 
 // _.propertyOf
 {
-    _.propertyOf({}); // $ExpectType (path: Many<PropertyName>) => any
-    _({}).propertyOf(); // $ExpectType LoDashImplicitWrapper<(path: Many<PropertyName>) => any>
-    _.chain({}).propertyOf(); // $ExpectType LoDashExplicitWrapper<(path: Many<PropertyName>) => any>
+    _.propertyOf({}) as (path: _.Many<_.PropertyName>) => any;
```

Comment:
These tests were also failing on the default branch - looks like TS 2.9 splits `PropertyName` into `string | number | symbol`, but earlier versions do not, so the `$ExpectType` does not work in this case.